content/_code-samples/freeze/py/check_freeze_status.py

```diff
@@ -0,0 +1,41 @@
+from xrpl.clients import JsonRpcClient
+from xrpl.models import AccountLines
+
+client = JsonRpcClient("https://xrplcluster.com")
+
+print("connected to mainnet")
+
+# Real accounts that were frozen on mainnet as an example
+
+# Issuer address
+issuer_addr = "rf1BiGeXwwQoi8Z2ueFYTEXSwuJYfV2Jpn"
+
+# Target address to query for freeze status
+target_addr = "rUpy3eEg8rqjqfUoLeBnZkscbKbFsKXC3v"
+
+token_name = "USD"
+
```

Comment:
I would add a print statement like this to indicate what you're trying to do when people run the snippet: 
```suggestion
print(f"searching for a trustline for {token_name} issued by {target_addr} for the address {issuer_addr}")
```